tests/sys.rs

```diff
@@ -64,6 +68,14 @@ async fn test_health(client: &impl Client) {
     assert!(resp.is_ok());
 }
 
+async fn test_start_initialization(client: &impl Client) {
+    let resp = sys::start_initialization(client, 1, 1).await.unwrap_err();
+    let ClientError::APIError { code, .. } = resp else {
+        panic!("must return an error because already initialized")
+    };
+    assert_eq!(code, 400);
+}
+
```

Comment:
if you start a new vault instance without the -dev parameter you should be able to initialise it with the variable.